proxy/envoy/config.go

```diff
@@ -472,6 +472,8 @@ func buildOutboundHTTPRoutes(mesh *proxyconfig.ProxyMeshConfig, sidecar proxy.No
 		}
 	}
 
+	addExternalTrafficVirtualHosts(&httpConfigs, config.EgressRules(), mesh)
```

Comment:
rename.. buildEgressHTTPRoutes

Reply:
Please follow the current conventions as found in other functions
buildEgressHTTPRoutes(mesh, sidecar .., config)